docs/static/releasenotes.asciidoc

```diff
@@ -12,6 +13,75 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-2-0]]
+=== Logstash 8.2.0 Release Notes
+
+60af28a0f - update google java format dependency (#13700) https://github.com/elastic/logstash/pull/13700
+43c657634 - Improve safety of stats retrieval from queue (#13942) https://github.com/elastic/logstash/pull/13942
+* Update failing policy in Central Management fetcher and license checker if hit ES down node https://github.com/elastic/logstash/pull/13689[#13689]
+* Print bundled jdk's version in launch scripts when `LS_JAVA_HOME` is provided https://github.com/elastic/logstash/pull/13880[#13880]
+* Update Jackson databind in ingest-converter tool https://github.com/elastic/logstash/pull/13900[#13900]
+* Fix/flaky error tests in missed retries to central management https://github.com/elastic/logstash/pull/13941[#13941]
+d2739a875 - Updates the LogstashService launcher to use the JDK specified (#13946) (4 weeks ago) <Andrea Selva>
```

Comment:
Looks like cleanup is still in progress, so I won't worry about this for now.  :-)